ephios/settings.py

```diff
@@ -160,6 +160,10 @@
 LOGIN_REDIRECT_URL = "/"
 PASSWORD_RESET_TIMEOUT = 28 * 24 * 60 * 60  # seconds
 
+# Store notifications for 14 days after creation
+NOTIFICATION_RETENTION_SECONDS = 14 * 24 * 60 * 60
+NOTIFICATION_SENDING_PROCESS_COUNT = 1 if DEBUG else 4
```

Comment:
should probably not have multiple processes try to send out emails using the same smtp server...